build/windows/installer/Package.en-us.wxl

```diff
@@ -0,0 +1,6 @@
+﻿<!--
+This file contains the declaration of all the localizable strings.
+-->
+<WixLocalization xmlns="http://wixtoolset.org/schemas/v4/wxl" Culture="en-US">
+    <String Id="DowngradeError" Value="A newer version of [ProductName] is already installed." />
```

Comment:
Is the installer replacing the `[ProductName]` with the binary's name, or should this file also have a `ToDo` comment?

Reply:
It gets replaced automatically :)